### PR TITLE
Adding guide for creating imagery layers

### PR DESCRIPTION
**Pull request by rhea-j**
Adding notebook guide to explain how we can create imagery using ArcGIS API for Python

cc - @AtmaMani 

-----

# Checklist

Please go through each entry in the below checklist and mark an 'X' if that condition has been met. Every entry should be marked with an 'X' to be get the Pull Request approved.

- [x] All `import`s are in the first cell? First block of imports are standard libraries, second block are 3rd party libraries, third block are all `arcgis` imports? Note that in some cases, for samples, it is a good idea to keep the imports next to where they are used, particularly for uncommonly used features that we want to highlight.
- [x] All `GIS` object instantiations are one of the following?
    - `gis = GIS()`
    - `gis = GIS('home')`
    - `gis = GIS(profile="your_online_portal")`
    - `gis = GIS('https://pythonapi.playground.esri.com/portal')`
    - `gis = GIS(profile="your_enterprise_portal")`
- [x] If this notebook requires setup or teardown, did you add the appropriate code to `./misc/setup.py` and/or `./misc/teardown.py`?
- [x] If this notebook references any portal items that need to be staged on AGOL/Python API playground, did you coordinate with a Python API team member to stage the item the correct way with the api\_data\_owner user?
- [x] Code refactored & split out across multiple cells, useful comments?
- [x] Consistent voice/tense/narrative style? Thoroughly checked for typos?
- [x] All images used like `` instead of ``? All map widgets contain a static image preview? (Call `mapview_inst.take_screenshot()` to do so)
- [x] All file paths are constructed in an OS-agnostic fashion with `os.path.join()`? (Instead of `r"\foo\bar"`, `os.path.join(os.path.sep, "foo", "bar")`, etc.)
- [x] **IF YOU WANT THIS SAMPLE TO BE DISPLAYED ON THE DEVELOPERS.ARCGIS.COM WEBSITE**, ping @ DavidJVitale so he can add it to the list for the next deploy

### CODE REVIEW

**Comment by rhea-j** [comment]
@BP-Ent Could you go through this guide?

File: guide/06-imagery-and-raster-analysis/creating_imagery_layers.ipynb [hunk 1]
```diff
@@ -0,0 +1,474 @@
+{
```
Comment (BP-Ent) [inline guide/06-imagery-and-raster-analysis/creating_imagery_layers.ipynb:1]: <p>For this functionality to work in ArcGIS Online, Azure library packages for Python (Azure SDK for Python - azure-storage-blob: 12.1&lt;= version &lt;=12.8) needs to be pre-installed. Refer here for more information <a href="https://docs.microsoft.com/en-us/azure/developer/python/azure-sdk-install" target="_blank">https://docs.microsoft.com/en-us/azure/developer/python/azure-sdk-install</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1036/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (BP-Ent) [inline guide/06-imagery-and-raster-analysis/creating_imagery_layers.ipynb:1]: <p>By using the <a href="https://developers.arcgis.com/python/api-reference/arcgis.raster.analytics.html#copy-raster" target="_blank">copy_raster</a> <strong>function</strong><em> </em>you can create one imagery layer by mosaicking multiple images.</p><p> </p><ul><li>Using the&nbsp;<code>raster_type_name</code>&nbsp;parameter, you can specify the raster type, which identifies metadata such as georeferencing, acquisition date, sensor type, and band wavelengths, along with a raster format. The supported raster types can be found in the&nbsp;<a href="https://app.reviewnb.com/Esri/arcgis-python-api/pull/1036/#Supported-raster-types" target="_blank">table</a> at the end of this guide. </li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1036/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (BP-Ent) [inline guide/06-imagery-and-raster-analysis/creating_imagery_layers.ipynb:1]: <p>Below are some examples for satellite imagery for the Landsat and Pleiades satellites along with the appropriate <code>raster_type_name</code> and <code>raster_type_params</code>:</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1036/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Comment (BP-Ent) [inline guide/06-imagery-and-raster-analysis/creating_imagery_layers.ipynb:1]: <p>The <a href="https://developers.arcgis.com/python/api-reference/arcgis.raster.analytics.html#copy-raster" target="_blank">copy_raster</a> function also enables you <strong>to</strong> create an imagery layer from just one image. This option supports all supported raster and image formats, multidimensional raster data, and configuration with various source types and compression settings. Use this option for simple imagery layers where advanced metadata is not needed.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1036/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Comment (BP-Ent) [inline guide/06-imagery-and-raster-analysis/creating_imagery_layers.ipynb:1]: <p>In this guide, you've learned how to create imagery layers from your local raster data or data in your datastore using the copy_raster and create_image_collection functions. These functions provide a simple way of creating an imagery layer from a normal raster dataset, multidimensional datasets, and even satellite datasets. You have also seen how to create an imagery layer from either a single or multiple raster datasets. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1036/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

**Review by BP-Ent — CHANGES_REQUESTED** [review]
Suggested changes made on reviewnb

**Comment by rhea-j** [comment]
@BP-Ent Thank you for going through it. I have made the changes as per your suggestions.

**Comment by rhea-j** [comment]
@priyankatuteja You had mentioned in the last call that we need to add the version from which the AGOL supports the functionality, but AGOL always moves forwards and users cant use an older version of it. So we don't need to specify the version. 

We have made all other changes.

**Comment by rhea-j** [comment]
@priyankatuteja @AtmaMani 
It will be great if we could merge this guide as well.

**Review by AtmaMani — APPROVED** [review]

**Pull request merged**